Header customizations in Version 8 and above (#7013)
Particular/docs.particular.net: nservicebus/recoverability/configure-error-handling.md

## Error message header customizations

Before a message is moved to the error queue it is possible to inspect and [error forwarding headers](/nservicebus/messaging/headers.md#error-forwarding-headers).

> [!WARNING]

> > Prior to Version 8 modifying existing headers on the failed message was also possible. Use a [recoverability pipeline behaviour](/nservicebus/recoverability/pipeline.md) to get full access to all headers. See the [upgrade guide](/nservicebus/upgrades/7to8/#header-manipulation-for-failed-messages) for more information.

Reviewer: ```suggestion
> Before Version 8, modifying existing headers on the failed message was possible. Starting version 8, use a [recoverability pipeline behavior](/nservicebus/recoverability/pipeline.md) to get full access to all headers. See the [upgrade guide](/nservicebus/upgrades/7to8/#header-manipulation-for-failed-messages) for more information.
```
